torch/distributed/pipeline/sync/checkpoint.py

```diff
@@ -208,7 +211,8 @@ def save_for_backward(self, *tensors: Tensor) -> None:  # pragma: no cover
 
 
 def save_rng_states(device: torch.device, rng_states: Deque[RNGStates],) -> None:
-    """:meth:`Checkpoint.forward` captures the current PyTorch's random number
+    """
+    Checkpoint.forward` captures the current PyTorch's random number
     generator states at CPU and GPU to reuse in :meth:`Recompute.backward`.
```

Comment:
This just has to be a one-liner:
```suggestion
    """:meth:`Checkpoint.forward` captures the current PyTorch's random number generator states at CPU and GPU to reuse in :meth:`Recompute.backward`.
```